app/assets/javascripts/app/account_transaction_report.coffee

```diff
@@ -0,0 +1,26 @@
+class AccountTransactionReport
+  constructor: (@$element) ->
+    return unless @$element.length > 0
+    @registerExportAllHandler()
+
+  registerExportAllHandler: ->
+    $('#js--exportAll').click (event) => @exportAllClicked(event)
```

Comment:
It seems a little weird to test on seeing if there is a `#table_billing`, then using `#js--exportAll` for the event. It almost seems like it would be cleaner to just do everything on `#js--exportAll`.